examples/browser/wdio.conf.js

```diff
@@ -5,9 +5,15 @@ const path = require('path');
 const port = 3000;
 const host = 'localhost';
 
+const debug = process.env.DEBUG;
+const spec = process.env.SPEC;
+const defaultSpecs = './test/**/*spec.ts';
+
 let server;
 
 exports.config = {
+    debug: true,
+    execArgv: ['--inspect=127.0.0.1:5859'],
```

Comment:
Why do you need?

Reply:
It doesn't work without that specific line for some reasons...